app/models/fingerprint.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: fingerprints
+#
+#  id         :integer          not null, primary key
+#  value      :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Fingerprint < ApplicationRecord
+  has_many :terms, dependent: :nullify
+
+  validates :value, uniqueness: true
+
+  alias_attribute :fingerprint_value, :value
```

Comment:
For now, I think I like the ergonomics of having it directly on the `Term` model. This may turn out to be something we remove later, which would be fine, but I'd rather deal with that down the road at this point.